From 7072e1239721b9d6891425f9e0d651865ec990a2 Mon Sep 17 00:00:00 2001
From: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Date: Wed, 23 Aug 2017 17:51:23 +0300
Subject: [PATCH 113/741] staging: fsl-dpaa2/mac: Comply with mc bus header
 updates

commit 0d3d455c7cf6e1ca5327453c4717ff2db8d278cd from
http://source.codeaurora.org/external/qoriq/qoriq-components/linux

The fsl-mc bus driver reorganized some of its headers, so
update our includes accordingly.

Signed-off-by: Ioana Radulescu <ruxandra.radulescu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/staging/fsl-dpaa2/mac/dpmac.c | 3 +--
 drivers/staging/fsl-dpaa2/mac/mac.c   | 1 -
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/drivers/staging/fsl-dpaa2/mac/dpmac.c b/drivers/staging/fsl-dpaa2/mac/dpmac.c
index f7827423329e..baef769834ab 100644
--- a/drivers/staging/fsl-dpaa2/mac/dpmac.c
+++ b/drivers/staging/fsl-dpaa2/mac/dpmac.c
@@ -29,8 +29,7 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
-#include "../../fsl-mc/include/mc-sys.h"
-#include "../../fsl-mc/include/mc-cmd.h"
+#include "../../fsl-mc/include/mc.h"
 #include "dpmac.h"
 #include "dpmac-cmd.h"
 
diff --git a/drivers/staging/fsl-dpaa2/mac/mac.c b/drivers/staging/fsl-dpaa2/mac/mac.c
index 98a73c5af005..1d8f07f7e453 100644
--- a/drivers/staging/fsl-dpaa2/mac/mac.c
+++ b/drivers/staging/fsl-dpaa2/mac/mac.c
@@ -47,7 +47,6 @@
 #include <linux/phy_fixed.h>
 
 #include "../../fsl-mc/include/mc.h"
-#include "../../fsl-mc/include/mc-sys.h"
 
 #include "dpmac.h"
 #include "dpmac-cmd.h"
-- 
2.17.1

